src/codemagic/tools/_app_store_connect/abstract_base_action.py

```diff
@@ -226,17 +281,31 @@ def list_builds(
         build_id: Optional[ResourceId] = None,
         pre_release_version: Optional[str] = None,
         processing_state: Optional[BuildProcessingState] = None,
+        beta_review_state: Optional[Union[BetaReviewState, Sequence[BetaReviewState]]] = None,
         build_version_number: Optional[int] = None,
         should_print: bool = True,
     ) -> List[Build]:
         from ..app_store_connect import AppStoreConnect
         _ = AppStoreConnect.list_builds  # Implementation
         raise NotImplementedError()
 
+    @abstractmethod
+    def list_review_submissions(
+        self,
+        application_id: ResourceId,
+        platform: Optional[Platform] = None,
+        review_submission_state: Optional[Union[ReviewSubmissionState, Sequence[ReviewSubmissionState]]] = None,
+        should_print: bool = True,
+    ) -> List[ReviewSubmission]:
+        from .action_groups import ReviewSubmissionsActionGroup
+        _ = ReviewSubmissionsActionGroup.list_review_submissions  # Implementation
+        raise NotImplementedError()
+
     @abstractmethod
     def submit_to_app_store(
         self,
         build_id: ResourceId,
+        cancel_previous_submissions: bool = False,
```

Comment:
To be a tiny bit more backwards compatible, better move this declaration after `max_build_processing_wait`.
Then previous non-kwarg calls such as `.submit_to_app_store(<build_id>, 10)` will remain working.